components/browser/menu/src/main/java/mozilla/components/browser/menu/item/BrowserMenuImageText.kt

```diff
@@ -56,25 +70,24 @@ open class BrowserMenuImageText(
     override fun bind(menu: BrowserMenu, view: View) {
         bindText(view)
 
-        bindImage(view)
-
         view.setOnClickListener {
             listener.invoke()
             menu.dismiss()
         }
     }
 
     private fun bindText(view: View) {
-        val textView = view.findViewById<TextView>(R.id.text)
-        textView.text = label
-        textView.setColorResource(textColorResource)
-    }
+        val textView = view.findViewById<TextView>(R.id.imageText)
+        with(textView) {
+            text = label
+            setColorResource(textColorResource)
 
-    private fun bindImage(view: View) {
-        val imageView = view.findViewById<AppCompatImageView>(R.id.image)
-        with(imageView) {
-            setImageResource(imageResource)
-            setTintResource(iconTintColorResource)
+            setCompoundDrawablesWithIntrinsicBounds(
```

Comment:
nit: I just added a KTX function for this 😃 
```suggestion
            putCompoundDrawablesWithIntrinsicBounds(start = tintedDrawable(imageResource, iconTintColorResource))
```